lib/block.ml

```diff
@@ -75,6 +75,7 @@ type t = {
   source_trees : string list;
   required_packages : string list;
   labels : Label.t list;
+  legacy_labels : Label.t list;
```

Comment:
How about making this a boolean and not allowing to mix both for the same block?

Reply:
done